<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonSettings</class>
 <widget class="QWidget" name="PythonSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,10,1">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Interpreter :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="interpreter"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Args :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="args"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="pdir_to_path">
       <property name="text">
        <string>Add Project dir to path</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="save_before">
       <property name="text">
        <string>Save files before run</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="settings">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_path">
      <attribute name="title">
       <string>Path</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="path">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_environment">
      <attribute name="title">
       <string>Environment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="environ"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/edit-add.svg</normaloff>:/app_data/icons/edit-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add element</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/remove.svg</normaloff>:/app_data/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Element</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/go-up.svg</normaloff>:/app_data/icons/go-up.svg</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/go-down.svg</normaloff>:/app_data/icons/go-down.svg</iconset>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
   <property name="toolTip">
    <string>Move Down</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
